pkg/api/query/v1.go

```diff
@@ -249,6 +250,20 @@ func (qapi *QueryAPI) parseStoreDebugMatchersParam(r *http.Request) (storeMatche
 	return storeMatchers, nil
 }
 
+func (qapi *QueryAPI) parseLookbackDeltaParam(r *http.Request) (time.Duration, *api.ApiError) {
```

Comment:
Could we also add the same parameter to the gRPC query API?

Reply:
done